pkg/virtual/framework/fixedgvs/register.go

```diff
@@ -51,8 +51,8 @@ func (vw *FixedGroupVersionsVirtualWorkspace) Register(vwName string, rootAPISer
 				StorageBuilders: make(map[string]func(apiGroupAPIServerConfig genericapiserver.CompletedConfig) (restStorage.Storage, error)),
 			},
 		}
-		for resourceName, builder := range restStorageBuilders {
-			cfg.ExtraConfig.StorageBuilders[resourceName] = builder
+		for resourceName, bdr := range restStorageBuilders {
```

Comment:
`s/bdr/restStorageBuilder` ?

I just got idea to look via k8s code base to match where we can

Reply:
sounds good, done